lib/transaction/input/multisigscripthash.js

```diff
@@ -62,6 +77,38 @@ MultiSigScriptHashInput.prototype._serializeSignatures = function() {
   });
 };
 
+MultiSigScriptHashInput.prototype.getScriptCode = function() {
+  var writer = new BufferWriter();
+  if (!this.redeemScript.hasCodeseparators()) {
+    var redeemScriptBuffer = this.redeemScript.toBuffer();
+    writer.writeVarintNum(redeemScriptBuffer.length);
+    writer.write(redeemScriptBuffer);
+  } else {
+    throw new Error('@TODO');
+  }
+  return writer.toBuffer();
+};
+
+MultiSigScriptHashInput.prototype.getSatoshisBuffer = function() {
```

Comment:
Maybe move this to input? It helps keep pay-to-script-hash/witness-scripthash handling more general